manticore/core/executor.py

```diff
@@ -48,29 +48,69 @@ def newFunction(self, *args, **kw):
 
 class Policy(object):
     ''' Base class for prioritization of state search '''
-    def __init__(self):
-        pass
-
-    def features(self, state):
-        ''' Save state features for prioritization before a state is stored '''
-        pass
+    def __init__(self, executor, *args, **kwargs):
+        super(Policy, self).__init__(*args, **kwargs)
+        self._executor = executor
+        self._executor.did_add_state += self._add_state_callback
+        
+    @contextmanager
+    def locked_context(self):
+        ''' Policy shared context dictionary '''
+        with self._executor.locked_context() as ctx:
+            policy_context = ctx.get('policy', None)
```

Comment:
I just tested and it will do that. As such, I think we should just use `get('policy')` as it's equivalent to `get('policy', None)`